caffe2/CMakeLists.txt

```diff
@@ -1076,26 +1076,9 @@ elseif(USE_CUDA)
         torch_cuda
     )
     if($ENV{ATEN_STATIC_CUDA})
-      if(CUDA_VERSION_MAJOR LESS_EQUAL 11)
-        target_link_libraries(torch_cuda_linalg PRIVATE
-            CUDA::cusolver_static
-            ${CUDAToolkit_LIBRARY_DIR}/liblapack_static.a     # needed for libcusolver_static
-        )
```

Comment:
Side note: We don't really test static_CUDA anymore, nor do we support CUDA-11 anymore, perhaps time to delete this logic completely...

Reply:
I'm not sure static CUDA is in use somewhere in META.. Could you propose to remove them?